libpod/options.go

```diff
@@ -304,6 +304,37 @@ func WithNamespace(ns string) RuntimeOption {
 	}
 }
 
+// WithPauseImage sets the pause image for libpod.
+// A pause image is used for inter-container kernel
+// namespace sharing within a pod. Typically, a pause
+// container is lightweight and is there to reap
+// zombie processes within its pid namespace.
+func WithPauseImage(img string) RuntimeOption {
```

Comment:
These should probably be WithDefaultPauseImage and WithDefaultPauseCommand - we'll want pod-specific ones to override the defaults